packages/protocol/contracts/MoxieBondingCurveV3.sol

```diff
@@ -914,11 +908,7 @@ contract MoxieBondingCurveV3 is IMoxieBondingCurveV3, SecurityModule {
     function updateGraduationMarketCap(uint32 _reserveRatio, uint256 _newGraduationMarketCap) external onlyRole(UPDATE_GRADUATION_MARKET_CAP_ROLE) {
         uint256 oldGraduationMarketCap = graduationMarketCap(_reserveRatio);
         graduationMarketCapOverrides[_reserveRatio] = _newGraduationMarketCap;
-        if (_newGraduationMarketCap == 0) {
-            emit GraduationMarketCapUpdated(_reserveRatio, oldGraduationMarketCap, defaultGraduationMarketCap, true);
-        } else {
-            emit GraduationMarketCapUpdated(_reserveRatio, oldGraduationMarketCap, _newGraduationMarketCap, false);
-        }
+        emit GraduationMarketCapUpdated(_reserveRatio, oldGraduationMarketCap, defaultGraduationMarketCap, _newGraduationMarketCap == 0);
```

Comment:
this event is incorrect because if the new graduation market cap is not 0 then it should emit `_newGraduationMarketCap` instead of `defaultGraduationMarketCap`